src/app/components/layout/footer/footer.tsx

```diff
@@ -9,8 +9,9 @@ export function Footer({ children, ...props }: HasChildren & FlexProps) {
       gap="space.05"
       p="space.05"
       bottom={0}
-      width="100vw"
-      maxWidth="100%"
+      // width="100vw"
+      // maxWidth="100%"
```

Comment:
You meaning to leave these commented out here?

Reply:
I most definitely am not, thanks! 

Apologies thats sloppy work from me, I'll fix it now

Reply:
I removed it now. This file is actually totally removed in https://github.com/leather-io/extension/pull/5796 but good to not add this stuff. 

Thanks for the review 👍